themes/helm/layouts/code/single.html

```diff
@@ -3,14 +3,13 @@
 
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-  <meta name="go-import" content="https://helm.sh{{substr .RelPermalink 0 -1}} git {{.Params.vanity}}">
-  <meta name="go-source" content="https://helm.sh{{substr .RelPermalink 0 -1}} {{.Params.vanity}} {{.Params.vanity}}/tree/master{/dir} {{.Params.vanity}}/blob/master{/dir}/{file}#L{line}">
-
-  <meta http-equiv="refresh" content="0; url={{.Params.vanity}}">
+  <meta name="go-import" content="https://helm.sh/{{.Params.Name}} git {{.Params.RepoURL}}">
```

Comment:
It looks like the parameters are camel-cased, but the params in helm.md weren't changed. Have you tested and confirmed this works?

Reply:
```
❯ curl -L http://localhost:1313/helm
<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="go-import" content="https://helm.sh/helm git https://github.com/helm/helm">
  <meta name="go-source" content="https://helm.sh/helm git https://github.com/helm/helm https://github.com/helm/helm/tree/dev-v3{/dir} https://github.com/helm/helm/blob/dev-v3{/dir}/{file}#L{line}">
  <meta http-equiv="refresh" content="0; url=https://github.com/helm/helm">
</head>

<body>
  Nothing to see here; <a href="https://github.com/helm/helm">move along</a>.
<script data-no-instant>document.write('<script src="/livereload.js?port=1313&mindelay=10"></' + 'script>')</script></body>

</html>
```